core/themes/seven/js/script.js

```diff
@@ -0,0 +1,292 @@
+/**
+ * @file
+ * Responsive Admin tabs.
+ */
+
+(function ($) {
+  "use strict";
+
+  Backdrop.behaviors.responsivePrimaryTabs = {
+    attach: function(context, settings) {
+      var $primaryTabs = $('ul.tabs.primary');
+      var $tabsWrapper = $($primaryTabs).parent();
+      var $tabs = $('li', $primaryTabs);
+      var responsiveTabs = false;
+      var previousWindowWidth;
+      var tabWidths = [];
+      var tabHeight;
+      var widestTabWidth = 0;
+      var expandControlWidth;
+      var activeTabNth = $('li.active', $primaryTabs).index();
+      var expandedTabsHeaderPadding = 0;
+      var defaultHeaderPadding = $('.l-header').css('padding-top');
+
+      // These are essentially breakpoints to be measured against the tabArea
```

Comment:
All comments throughout this file are missing trailing periods (required per our code standards).